peers/discv5.go

```diff
@@ -95,3 +98,10 @@ func (d *DiscV5) Discover(topic string, period <-chan time.Duration, found chan<
 	d.net.SearchTopic(discv5.Topic(topic), period, found, lookup)
 	return nil
 }
+
+// InsertNodes inserts givens peers as nodes into Discovery V5 table.
+func (d *DiscV5) InsertNodes(topic discv5.Topic, nodes ...*discv5.Node) {
```

Comment:
just for info - in rendezvous node have to be registered every defined time period (i think 10s - 20s atm), after this time period it is purged from database. the reason for doing so is to guarantee better peer liveness.